migration/incremental/025-tag-map-storage-type.sql

```diff
@@ -0,0 +1,19 @@
+-- tag_v
+UPDATE pg_catalog.pg_type AS t 
```

Comment:
Any reason to do this over a plain
```SQL
alter type tag_map set (storage = extended);
```

Reply:
Alter type is not supported on pg12
<img src="https://user-images.githubusercontent.com/222471/170989051-41767ab8-99de-4046-8468-47e5bd8368f2.jpg" width=300px>

Reply:
Are you opposed to making the current code an exception for v12, rather than the default behavior?

Reply:
I don't see the benefit of added branching, but it surely increases complexity.

Reply:
I find it hard to agree that manually updating catalog tables reduces complexity.

Reply:
I would agree if this line was gone entirely, but it's "this complex line" or "alter type". Which is in my book strictly more complex than just "this complex line".

Reply:
I'd agree if at one point pg12 won't be gone. And once it is gone we can go ahead and remove all such branches given we can find them.